### ISSUE TITLE
Bootstrap3 Modal + form /x post from Select2

### ISSUE THREAD

**Issue by bringking** [angular-ui/ui-select2#171]
Has anyone run into issues using select2 inside a Bootstrap 3 modal window? Using a standard modal with -

I am using pretty standard bootstrap markup ->

```
<div class="modal-header">
    <h3>Edit </h3>
</div>
<div class="modal-body">
    <form>
        <div class="row">
            <div class="col-md-6 form-group">
                <div>
                    <label></label>
                    <input class="form-control" type="text"/>
                </div>
            </div>
           <div class="col-md-6 form-group">
          <label></label>
         <select  ui-select2  class="form-control"  data-placeholder="Pick a {{selectFromSchema.target}}" ng-model="selectionModel" ng-options="guessLabel(item) for item in items">\
                           </select>
            </div>
        </div>
    </form>
</div>
<div class="modal-footer">
    <button class="btn btn-primary" >Save</button>
    <button class="btn btn-default">Cancel</button>
</div>
```

and getting results like so-

![select2grab](https://f.cloud.github.com/assets/864845/2380115/0630e006-a8a5-11e3-84f0-3acb21fc710c.png)

Any ideas?

**Comment by lucnap** [comment]
I have the same problem, please how you have solved the problem?

**Comment by plentz** [comment]
@lucnap @bringking have you guys loaded select2-bootstrap.css?

**Comment by bringking** [comment]
My issues have been resolved using bootstrap latest and ui-select2 0.0.5.
